### PR TITLE
Add conversions from wrapper tasks to continuous/finite tasks

### PR DESCRIPTION
**Pull request by shadaj**

### CODE REVIEW

**Comment by shadaj** [comment]
Ping @Paxelord

File: commons/src/main/scala/com/lynbrookrobotics/potassium/commons/flywheel/Flywheel.scala [hunk 1]
```diff
@@ -38,23 +38,6 @@ abstract class Flywheel {
   }
 
   object velocityTasks {
-    class WaitForVelocity(vel: Signal[Frequency], tolerance: Frequency)
```
Comment (PhilipAxelrod) [inline commons/src/main/scala/com/lynbrookrobotics/potassium/commons/flywheel/Flywheel.scala:41]: Wait, I don't get what is replacing this?

**Review by PhilipAxelrod — CHANGES_REQUESTED** [review]

File: commons/src/main/scala/com/lynbrookrobotics/potassium/commons/flywheel/Flywheel.scala (hunk 1 above)
Comment (shadaj) [reply commons/src/main/scala/com/lynbrookrobotics/potassium/commons/flywheel/Flywheel.scala:41]: `WhileAtVelocity.toFinite`

Comment (PhilipAxelrod) [reply commons/src/main/scala/com/lynbrookrobotics/potassium/commons/flywheel/Flywheel.scala:41]: Ok cool

**Review by PhilipAxelrod — APPROVED** [review]

**Pull request merged**